generator/java/build.gradle

```diff
@@ -33,13 +36,21 @@ task generateClient(type: GradleBuild) {
     def taskList = ['cleanPreviousJavaBuild'];
 
 	rootProject.swaggerSourceList.each {
-        def apiName = it.getName().replace(".json", "").toLowerCase();
         def jsonFile = file(it.path);
         def parsedJson = new groovy.json.JsonSlurper().parseText(jsonFile.text);
```

Comment:
from my understanding we only parse the json to get the API version, but as we now have it from the swagger file name, do we still need to do it?